apps/drupal/drupal.conf

```diff
@@ -130,93 +163,42 @@ location / {
         open_file_cache_min_uses 2;
         open_file_cache_errors off;
     }
+    location ~* \.(woff)$ {
+         add_header "Access-Control-Allow-Origin" "*";
+    }
 
     ## PDFs and powerpoint files handling.
-    location ~* ^.+\.(?:pdf|pptx?)$ {
-        expires 30d;
+    ##location ~* ^.+\.(?:pdf|pptx?)$ {
+    ##    expires 30d;
         ## No need to bleed constant updates. Send the all shebang in one
         ## fell swoop.
-        tcp_nodelay off;
-    }
+    ##    tcp_nodelay off;
+    ##}
 
-    ## MP3 and Ogg/Vorbis files are served using AIO when supported. Your OS must support it.
-    location ^~ /sites/default/files/audio/mp3 {
-        location ~* ^/sites/default/files/audio/mp3/.*\.mp3$ {
-            directio 4k; # for XFS
-            ## If you're using ext3 or similar uncomment the line below and comment the above.
-            #directio 512; # for ext3 or similar (block alignments)
-            tcp_nopush off;
-            aio on;
-            output_buffers 1 2M;
-        }
-    }
-
-    location ^~ /sites/default/files/audio/ogg {
-        location ~* ^/sites/default/files/audio/ogg/.*\.ogg$ {
-            directio 4k; # for XFS
-            ## If you're using ext3 or similar uncomment the line below and comment the above.
-            #directio 512; # for ext3 or similar (block alignments)
-            tcp_nopush off;
-            aio on;
-            output_buffers 1 2M;
-        }
-    }
-
-    ## Pseudo streaming of FLV files:
-    ## http://wiki.nginx.org/HttpFlvStreamModule.
-    ## If pseudo streaming isn't working, try to comment
-    ## out in nginx.conf line with:
-    ## add_header X-Frame-Options SAMEORIGIN;
-    location ^~ /sites/default/files/video/flv {
-        location ~* ^/sites/default/files/video/flv/.*\.flv$ {
-            flv;
-        }
-    }
-
-    ## Pseudo streaming of H264/AAC files. This requires an Nginx
-    ## version greater or equal to 1.0.7 for the stable branch and
-    ## greater or equal to 1.1.3 for the development branch.
-    ## Cf. http://nginx.org/en/docs/http/ngx_http_mp4_module.html.
-    location ^~ /sites/default/files/video/mp4 { # videos
-        location ~* ^/sites/default/files/video/mp4/.*\.(?:mp4|mov)$ {
-            mp4;
-            mp4_buffer_size 1M;
-            mp4_max_buffer_size 5M;
-        }
-    }
-
-    location ^~ /sites/default/files/audio/m4a { # audios
-        location ~* ^/sites/default/files/audio/m4a/.*\.m4a$ {
-            mp4;
-            mp4_buffer_size 1M;
-            mp4_max_buffer_size 5M;
-        }
-    }
-
-    ## Advanced Help module makes each module provided README available.
-    location ^~ /help/ {
```

Comment:
@debojitcampd 

Is this required to to removed? I think advanced help module will not work.